swap_meet/vendor.py

```diff
@@ -1,2 +1,87 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory=[]
+        self.inventory=inventory
+    
+    def add(self,new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, removed_item):
+        if removed_item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(removed_item)
+            return removed_item
+
+    def get_by_category(self, category):
+        item_list=[]
+        for item in self.inventory:
+            if item.category == category:
+                item_list.append(item)
+
+        return item_list
+
+    def swap_items(self, friend, my_item, their_item):
+        if my_item in self.inventory:
+            if their_item in friend.inventory:
```

Comment:
Chained `if` statements like this can be combined with `and`:
```suggestion
        if my_item in self.inventory and their_item in friend.inventory:
```